### PR TITLE
Popup: add overflow menu in toolbar

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: MVCDemos/Views/Popup/Overview.cshtml [hunk 1]
```diff
@@ -15,6 +15,24 @@
...
         .ShowCloseButton(false)
         .ToolbarItems(barItems => {
+            barItems.Add()
+                .Toolbar(Toolbar.Top)
+                .LocateInMenu(ToolbarItemLocateInMenuMode.Always)
+                .Widget(widget => widget.Button()
+                    .Text("More info")
+                    .OnClick(@<text>
+                        function showInfo(data) {
+                            const message = `More info abou ${currentEmployee.FirstName} ${currentEmployee.LastName}`;
```
Comment (EugeniyKiyashko) [inline MVCDemos/Views/Popup/Overview.cshtml]: typo

File: JSDemos/Demos/Popup/Overview/React/App.js [hunk 2]
```diff
@@ -19,6 +21,10 @@ class App extends React.Component {
     };
     this.showInfo = this.showInfo.bind(this);
     this.hideInfo = this.hideInfo.bind(this);
+    this.moreInfoButtonOptions = {
+      text: 'More info',
+      onClick: this.lookMoreInfo.bind(this),
```
Comment (EugeniyKiyashko) [inline JSDemos/Demos/Popup/Overview/React/App.js]: Let's rename `lookMoreInfo` into `showMoreInfo`

Comment (marker-dao) [reply JSDemos/Demos/Popup/Overview/React/App.js]: Done

File: MVCDemos/Views/Popup/Overview.cshtml (hunk 1 above)
Comment (marker-dao) [reply MVCDemos/Views/Popup/Overview.cshtml]: Done

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**